### PR TITLE
raise phpstan level to 5

### PR DESCRIPTION
**Pull request by garak**

### CODE REVIEW

**Comment by JellyBellyDev** [comment]
@garak, in wip, right?

**Comment by garak** [comment]
> @garak, in wip, right?

well. it's done indeed until level 6
I tried to go further, but level 7 i a bit complex, so I gave up for now. Maybe in the future...

**Comment by garak** [comment]
Never mind: the meant level was 5, but I forgot the "6" in config after trying to upgrade.
Now it should be fine 😅

File: composer.json [hunk 1]
```diff
@@ -101,5 +101,11 @@
...
         }
-    }
+    },
+    "funding": [
+        {
+            "type": "paypal",
+            "url": "https://paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=ETT4JRJARLTSC"
+        }
+    ]
 }
```
Comment (JellyBellyDev) [inline composer.json]: already on master. How come we apply the change again?

Comment (garak) [reply composer.json]: I'll rebase

Comment (garak) [reply composer.json]: done

**Pull request merged**